pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -120,6 +124,15 @@ func (c *Cluster) Start() error {
 		return errors.Wrap(err, "failed to check for mgrs to skip reconcile")
 	}
 
+	shouldRotateKeys, err := shouldRotateMgrKeys(c.context, c.clusterInfo)
```

Comment:
directly assign it to `c.shouldRotateCephxKeys` so that we can avoid declaring more more variable?